### DISCUSSION TITLE
writing indices in remote filesystem

### DISCUSSION THREAD

**Discussion by kyogesha** [microsoft/hyperspace#462] [Q&A]
Hi, 
Is it possible to write indices in remote filesystem like s3 (possibly in future)?

**Comment by sezruby** [comment]
Though I haven't tested yet with a s3 path, you can change the path for index data using
```
spark.conf.set("spark.hyperspace.system.path", "/path/to/indexes")
```

Then your index (`indexName`) will be written in `/path/to/indexes/indexName` at index creation.
Note that Hyperspace tries to read indexes from the path when query time, so it's not possible to use multiple index data paths for now.
